client/src/partials/debtor_groups/debtor_groups.html

```diff
@@ -0,0 +1,281 @@
+                  </div>
+                </div>
+
+                <!-- Find an account for debtor group -->
+                <div class="form-group" ng-class="{ 'has-error' : ActionForm.$submitted && ActionForm.account.$invalid }">
+                  <label class="control-label">
+                    {{ "COLUMNS.ACCOUNT" | translate }}
+                  </label>
+                  <input
+                    class="form-control"
+                    name="account"
+                    placeholder="{{ 'PLACEHOLDERS.ENTER.ACCOUNT' | translate }}..."
+                    uib-typeahead="account.id as account.account_number for account in DebtorGroupCtrl.accounts.data | filter:$viewValue | limitTo:8" class="form-control"
+                    typeahead-template-url="partials/templates/typeahead/account.html"
+                    ng-model="DebtorGroupCtrl.debtorGroup.account_id" required>
+                  <div class="help-block" ng-messages="ActionForm.account.$error" ng-show="ActionForm.$submitted">
+                    <div ng-messages-include="partials/templates/messages.tmpl.html"></div>
+                  </div>
+                </div>
+
+                <!-- Find price list for the group -->
+                <div class="form-group" ng-class="{ 'has-error' : ActionForm.$submitted && ActionForm.price_list.$invalid }">
+                  <label class="control-label">
+                    {{ "COLUMNS.PRICE_LIST" | translate }}
+                  </label>
+                  <select
+                    class="form-control"
+                    name="price_list"
+                    ng-model="DebtorGroupCtrl.debtorGroup.price_list_uuid"
+                    ng-options="price.uuid as price.label for price in DebtorGroupCtrl.prices.data"
+                    required>
+                    <option value="" disabled>{{ "PLACEHOLDERS.SELECT.PRICE_LIST" | translate }}</option>
+                  </select>
+                  <div class="help-block" ng-messages="ActionForm.account.$error" ng-show="ActionForm.$submitted">
+                    <div ng-messages-include="partials/templates/messages.tmpl.html"></div>
+                  </div>
+                </div>
+
+                <!-- Max credit for the debtor group -->
+                <div class="form-group">
+                  <label class="control-label">
+                    {{ "COLUMNS.MAX_CREDIT" | translate }}
+                  </label>
+                  <input
+                    type="number"
+                    class="form-control"
+                    name="max_credit"
+                    placeholder="{{ 'COLUMNS.MAX_CREDIT' | translate }}..."
+                    ng-model="DebtorGroupCtrl.debtorGroup.max_credit">
+                </div>
+
+              </fieldset>
+
+              <!-- caracteristics -->
+              <fieldset>
+                <legend>{{ 'COLUMNS.CARACTERISTICS' | translate }}</legend>
+                <div class="form-group col-xs-6">
```

Comment:
In the future, the proper way to make a checkbox input in bootstrap is like this:
```html
  <div class="checkbox">
    <label>
      <input type="checkbox"> Check me out
    </label>
  </div>
```
It should not be wrapped in a `class="form-group"`, or else it looks very strange.  [Check out the source](http://getbootstrap.com/css/#forms-example).